code/modules/library/book.dm

```diff
@@ -361,7 +361,7 @@
 
 /obj/item/book/random/Initialize()
 	..()
-	var/list/books = GLOB.library_catalog.get_random_book(amount)
+	var/list/books = GLOB.library_catalog.get_random_book(amount, FALSE)
```

Comment:
By default, all calls that use this proc have it not run async, can you change `doAsync = TRUE` to false in the proc definition?
```suggestion
	var/list/books = GLOB.library_catalog.get_random_book(amount, FALSE)
```

Reply:
It already is

`/datum/library_catalog/proc/get_random_book(amount = 1, doAsync = TRUE)`

Reply:
Nah he wants you to change the TRUE there to FALSE since all calls of this proc set it to `FALSE`, but this isn't really in scope so